ERROR - 2024-07-15 14:31:49 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs24_ciudadanos', 3306, '', 0)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-07-15 14:31:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'fs24_ciudadanos'
[Method: GET, Route: ver_regiones]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `regiones`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Regiones->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-15 14:34:43 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs24_ciudadanos', 3306, '', 0)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-07-15 14:34:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'fs24_ciudadanos'
[Method: GET, Route: ver_regiones]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `regiones`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Regiones->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-15 14:34:43 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs24_ciudadanos', 3306, '', 0)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-07-15 14:34:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'fs24_ciudadanos'
[Method: GET, Route: ver_regiones]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `regiones`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Regiones->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-15 14:34:48 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs24_ciudadanos', 3306, '', 0)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'fs24_ciudadanos' in C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\fs2024\ci4_ciudadanos\app\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(933): App\Controllers\Regiones->index()
#7 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
#8 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fs2024\ci4_ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\fs2024\ci4_ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-07-15 14:34:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'fs24_ciudadanos'
[Method: GET, Route: ver_regiones]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `regiones`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Regiones.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Regiones->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regiones))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-15 14:35:47 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "nivelesacademicos.php"
[Method: GET, Route: ver_niveles_academicos]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('nivelesacademicos.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('nivelesacademicos', [], true)
 3 APPPATH\Controllers\NivelesAcademicos.php(15): view('nivelesacademicos', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\NivelesAcademicos->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NivelesAcademicos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-15 14:42:31 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "nivelesacademicos.php"
[Method: GET, Route: ver_niveles_academicos]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('nivelesacademicos.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('nivelesacademicos', [], true)
 3 APPPATH\Controllers\NivelesAcademicos.php(15): view('nivelesacademicos', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\NivelesAcademicos->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NivelesAcademicos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-15 14:43:12 --> ErrorException: Undefined variable $region
[Method: GET, Route: ver_niveles_academicos]
in APPPATH\Views\niveles_academicos.php on line 25.
 1 APPPATH\Views\niveles_academicos.php(25): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $region', 'C:\\xampp\\htdocs\\fs2024\\ci4_ciudadanos\\app\\Views\\niveles_academicos.php', 25)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\fs2024\\ci4_ciudadanos\\app\\Views\\niveles_academicos.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('niveles_academicos', [], true)
 5 APPPATH\Controllers\NivelesAcademicos.php(15): view('niveles_academicos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\NivelesAcademicos->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NivelesAcademicos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-15 15:00:18 --> Error: Class "App\Controllers\MunicipioCiudadanosModelsModel" not found
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
